dwellingly/src/views/addProperty.js

```diff
@@ -0,0 +1,162 @@
+import React, { Component } from 'react';
+import { Form, Field, Formik } from 'formik';
+import * as Yup from 'yup';
+import { UserContext } from '../App';
+import * as axios from 'axios';
+
+const validationSchema = Yup.object().shape({
+    name: Yup.string()
+        .min(5, "*Name must have at least 2 characters")
+        .max(50, "*Names can't be longer than 100 characters")
+        .required("*Name is required"),
+    address: Yup.string()
+        .required("*Address is required"),
+    city: Yup.string()
+        .required("*City is required"),
+    zipcode: Yup.string()
+        .required("*Zipcode is required"),
+    state: Yup.string()
+        .required("*State is required")
```

Comment:
We'll want to keep in mind in the future a general "Address schema" that we can check forms against to make sure we have common types for things like state (OR or Oregon?), Zip (5 digits? or 5 digits + the postal extension?), etc.
Not sure what that will look like yet so we don't need to worry about it here